AppCenterAnalytics/AppCenterAnalyticsTests/MSAnalyticsTransmissionTargetTests.m

```diff
@@ -2,9 +2,11 @@
 
```

Comment:
Can you remove XCTest import even though it isn't your change? XCTest is imported in MSTestFrameworks.h.